src/app/bav/controllers/confirmDetails.js

```diff
@@ -38,11 +38,12 @@ class ConfirmDetailsController extends BaseController {
 
   async saveBavData(axios, bavData, req) {
     const headers = {
-      "x-govuk-signin-session-id": req.session.tokenId,
+      "x-govuk-signin-session-id": req.session,
```

Comment:
Is this change right? I thought it was always `req.session.tokenId`?

Reply:
No sorry that was an accidental deletion